modules/help_editor/php/help_editor.class.inc

```diff
@@ -174,7 +131,7 @@ class Help_Editor extends \NDB_Menu_Filter
         return array_merge(
             $deps,
             array(
-             $baseURL . "/help_editor/js/help_editor_helper.js",
+             $baseURL . "/help_editor/js/help_editor.js",
              $baseURL . "/help_editor/js/columnFormatter.js",
```

Comment:
Regardless of whether `columnFormatter` is in its own file or not, I don't think it needs to be included here because it's imported by your helpEditor.jsx.